packages/composer/README.md

```diff
@@ -0,0 +1,33 @@
+# `Metadata Composer`
+
+> A library that merges metadata from subgraphs indexing different networks. The interface of this library is closely similar to the `theGraph` interface (i.e. input/output) so that if dApps needs metadata from one network, it can simply use `theGraph` api with minimal changes.
+
+## Usage
+```
+import { MetadataComposer } from '@clrfund/metadata-composer'
+
+const urls = ['https://api.thegraph.com/subgraphs/name/yuetloo/metadata-rinkeby']
+const composer = new MetadataComposer(urls)
+
+const query = '{ metadataEntries{id} }'
+const result = await composer.query(query)
```

Comment:
return FE friendly responses, type out the response to make error handling explicit
ie. `const { data, loading, error } = await composer.query(query)`

Reply:
done.